client/src/Components/Spectrogram/SettingsPane.jsx

```diff
@@ -5,19 +5,12 @@
 import React, { useEffect, useState } from 'react';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faArrowRight } from '@fortawesome/free-solid-svg-icons';
```

Comment:
removing all these bootstrap imports must've been satisfying <3

Reply:
It really was :D